### PR TITLE
Remove deprecated Blade Ally effects and broken links

### PR DESCRIPTION
**Pull request by roxim5**

### CODE REVIEW

**Comment by TikaelSol** [comment]
Delay until migration can be written

**Comment by stwlam** [comment]
Conflicts!

**Comment by roxim5** [comment]
I'll deal with them here when I have time... hopefully later tonight. Otherwise tomorrow.

**Comment by roxim5** [comment]
> Conflicts!

Jobs done

File: src/module/migration/migrations/822-blade-ally-consolidation.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                .replace(/\bfeat-effects\.rGSc2PtvU3mgm18S\b/g, "equipment-srd.gmMrJREf4JSHd2dZ")
+                .replace(/\bfeat-effects\.Effect: Blade Ally Ghost Touch Rune\b/g, "equipment-srd.Ghost Touch")
+                .replace(/\bfeat-effects\.Rgio0hasm2epEMfh\b/g, "equipment-srd.JQdwHECogcTzdd8R")
+                .replace(
+                    /\bfeat-effects\.Effect: Blade Ally Disrupting Rune (Greater)\b/g,
+                    "equipment-srd.Disrupting (Greater)"
+                )
+                .replace(/\bfeat-effects\.HjfIXg5btodThCTW\b/g, "equipment-srd.oVrVzML63VFvVfKk");
+
+        source.system.description = recursiveReplaceString(source.system.description, rename);
```
Comment (stwlam) [inline src/module/migration/migrations/822-blade-ally-consolidation.ts:36]: `source.system.description` only has one property

**Pull request merged**